app/src/main/java/org/wikipedia/page/leadimages/LeadImagesHandler.java

```diff
@@ -243,16 +245,26 @@ public void onCallToActionClicked() {
         });
     }
 
-    public void openImageInGallery(@Nullable  String language) {
+    public void openImageInGallery(@Nullable String language) {
         if (getPage() != null && isLeadImageEnabled()) {
             String imageName = getPage().getPageProperties().getLeadImageName();
             if (imageName != null) {
                 String filename = "File:" + imageName;
                 WikiSite wiki = language == null ? getTitle().getWikiSite() : WikiSite.forLanguageCode(language);
+
+                JavaScriptActionHandler.ImageHitInfo hitInfo = new JavaScriptActionHandler.ImageHitInfo(pageHeaderView.image.getLeft(),
```

Comment:
If the image was face detected (e.g. Barack Obama), the image will be cropped and it looks slightly weird during the transition.

Reply:
Yes, that's a bit of an issue, but I'm not sure I have a solution yet. It could be some technical debt for the future.